lib/core/comments/comments.module.ts

```diff
@@ -34,9 +38,13 @@ import { CommentsComponent } from './comments.component';
         DataTableModule,
         FormsModule,
         ReactiveFormsModule,
```

Comment:
Can you verify if we need these modules? I'm not sure that we need them

- `ReactiveFormsModule`
- `DataTableModule`
- `DataColumnModule`

Reply:
The only module that is necessary is the **FormsModule**. Without it, the _mat-form-field_ throws an error. I'll delete the rest 😄
![image](https://user-images.githubusercontent.com/64163335/182783382-d1c74a4e-3c4e-4886-b466-d37ded4ee9b7.png)